xml/cl.xml

```diff
@@ -1357,8 +1357,9 @@ server's OpenCL/api-docs repository.
 
     <enums name="cl_command_buffer_state_khr" vendor="Khronos">
         <enum value="0"            name="CL_COMMAND_BUFFER_STATE_RECORDING_KHR"/>
-        <enum value="1"            name="CL_COMMAND_BUFFER_STATE_EXECUTABLE_KHR"/>
-        <enum value="2"            name="CL_COMMAND_BUFFER_STATE_PENDING_KHR"/>
+        <enum value="1"            name="CL_COMMAND_BUFFER_STATE_FINALIZED_KHR"/>
+        <enum value="2"            name="CL_COMMAND_BUFFER_STATE_EXECUTABLE_KHR"/>
```

Comment:
Changing the value of enums could break existing user code, lets add `CL_COMMAND_BUFFER_STATE_FINALIZED_KHR` to the end.